README.md

```diff
@@ -1,216 +1,265 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Mobile Application
+# Compose Multiplatform mobile application
 
-Use this template to start developing your own Compose Multiplatform application targeting Android and iOS (Alpha).
+> Compose Multiplatform is in Alpha. It may change incompatibly and require manual migration in the future.
+
+You can use this template to start developing your own [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) application.
+The result will be a Kotlin Multiplatform project targeting Android and iOS.
 
 ![](readme_images/banner.png)
 
-## Setting up your development environment
+## Set up an environment
 
-Your Compose Multiplatform application targeting Android and iOS is a Kotlin Multiplatform project.
```

Comment:
I think this sentence is important because we're targeting people who come here having heard about Compose Multiplatform, so we need to estalblish that we're building on top of KMP. Unless this is mentioned somewhere else in the doc, I think this sentence should stay

Reply:
I tried to include it in the README introduction

Reply:
LGTM.